Wikipedia/Code/ViewControllerRouter.swift

```diff
@@ -17,6 +17,20 @@ class ViewControllerRouter: NSObject {
             completion()
             return false
         }
+        
+        let theme = appViewController.theme
+        
+        let present = { (viewController: UIViewController) in
```

Comment:
maybe rename to something like presentOrPush? I feel like it's unclear that this could possibly push.

Reply:
Good idea, fixed!